lib/deaf_grandma.rb

```diff
@@ -35,6 +50,18 @@ def get_user_input
     gets.chomp
   end
 
+  def soft
```

Comment:
Generally, you want your method names to be verbs - something an instance of the class in question could *do*.

Reply:
^^Maybe that's not right.

Better: you want your method names to be either verbs (something an instance of the class in question could *do*) -- or else you want your method names to be nouns - a piece of data an instance of the class can tell you about itself